charmcraft.yaml

```diff
@@ -92,4 +90,20 @@ requires:
   ingress:
     interface: traefik_route
     limit: 1
-
+config:
+  options:
+    custom-config-repository:
+      type: string
+      description: >
+        The repository URL where the Wazuh configuration is stored. The username has to be provided as in 
+        git+ssh://username@repository@branch, where the branch is optional.
+        The charms expects the repository to contain a /var/ossec directory to be copied over /var/ossec on
```

Comment:
I'd suggest dropping the first "/". "The charms expects the repository to contain a var/ossec directory to be copied over /var/ossec"